crates/fluvio-connector-derive/src/generator.rs

```diff
@@ -119,7 +119,11 @@ fn init_and_parse_config(config_type_path: &Path) -> TokenStream {
         let common_config = ::fluvio_connector_common::config::ConnectorConfig::from_value(config_value.clone())?;
         ::fluvio_connector_common::tracing::debug!("{:#?}", common_config);
 
```

Comment:
can we update unit test for derive macro?

Reply:
not sure how we could test this from the UI tests. I will add a unit test to fluvio_connector_common::resolve_config tho

Reply:
Yes, let's add to fluvio_connector_common::resolve_config

Reply:
I added a test to it on `crates/fluvio-connector-package/src/lib.rs` fluvio_connector common` is re-exporting the function